lang/wasm.cc

```diff
@@ -6,8 +6,6 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-#ifdef SCYLLA_ENABLE_WASMTIME
-
```

Comment:
Please check that the series continues building without wasmtime.

Reply:
Wasmtime is now downloaded as part of the compilation of the .rs files, so it should never be missing, but it may not work correctly in ARM (it didn't work when we used libwasmtime.a) - I'll try to check this soon. If it doesn't work with ARM I'll bring the `SCYLLA_ENABLE_WASMTIME` back

Reply:
Wait what? Didn't we add it to the toolchain?

In any case it may not be available on all architectures.

Reply:
We did add Rust to the toolchain, but we didn't add all the crates that will be used in this patch.
Actually, we should probably discuss how do we want to deal with downloading the new crates.
Currently, the dependency will be downloaded if it's missing during the compilation (it won't be re-downloaded if it's already present).
If we don't want that, we could update the toolchain every time a dependency is changed.
We can also update the toolchain only periodically, and some dependencies could ever need to be downloaded during compilation.

I didn't really see benefits to the latter options because after the first build, there basically is no difference between them

Reply:
Actually, where do we want to keep the downloaded crates? Cargo uses `$HOME/.cargo` by default.
To test if everything works with ARM [this](https://github.com/scylladb/scylla-pkg/issues/3062) would help a lot